tools/cmake_test.py

```diff
@@ -194,10 +194,14 @@ def __init__(self, prepare_command, post_command, binary, repeat,
 
         if "rpunit" in binary or "rpfixture" in binary:
             unit_args = [
-                "--overprovisioned", "--unsafe-bypass-fsync 1",
-                "--default-log-level=trace", "--logger-log-level='io=debug'",
+                "--unsafe-bypass-fsync 1", "--default-log-level=trace",
+                "--logger-log-level='io=debug'",
                 "--logger-log-level='exception=debug'"
             ] + COMMON_TEST_ARGS
+
+            if self.ci:
```

Comment:
nit: this commit would make more sense if it was after the commit where the `ci` property is introduced.

Reply:
Will fix in re-push.